sources/logdog/conf/logdog.aws-k8s-1.21-nvidia.conf

```diff
@@ -0,0 +1 @@
+aws-k8s.conf
```

Comment:
As we discussed offline, I removed the `nvidia-persistenced` daemon from this PR, since it will try to look for devices under `/dev`, which take a while to be created by the kernel. We need to figure out a way to make the daemon wait or be triggered once the devices are created (a.k.a `Wants=nvidia0.device`).